en-US/snippets/2017/mar2017.lang

```diff
@@ -0,0 +1,87 @@
+## NOTE: reference bug 1336204; Deadline: 24 February, 2017
+## NOTE: Guidelines on localizing snippets: https://goo.gl/PluA97.  The tone is QUAC (Quick, Untamed, Alert, Clever). Do your best to make it local and culturally relevant.
+## NOTE: Link for Google Play: https://play.google.com/store/apps/details?id=org.mozilla.firefox&amp
+## NOTE: Link for App Store: https://itunes.apple.com/app/apple-store/id989804926?mt=8
+## NOTE: Campaign: Mobile Promotions
+
+
+## MAX_LENGTH: 120 characters
```

Comment:
I can remove this one (120) consider the current length. When I asked about the character limitation is applied to all or just the longer ones.  He said the short ones are "headers" and should be kept at 120, so the two lines are two separate entries.  I only noted in the longer ones.  But even these are so short, that they can't easily reach the limit.  I can skip the short ones and only note the long (full text) to the limit of 180.